editor/src/messages/input_mapper/default_mapping.rs

```diff
@@ -177,6 +177,7 @@ pub fn default_mapping() -> Mapping {
 		entry!(KeyDown(Lmb); action_dispatch=PathToolMessage::DragStart { add_to_selection: Shift }),
 		entry!(PointerMove; refresh_keys=[Alt, Shift], action_dispatch=PathToolMessage::PointerMove { alt_mirror_angle: Alt, shift_mirror_distance: Shift }),
 		entry!(KeyDown(Delete); action_dispatch=PathToolMessage::Delete),
+		entry!(KeyDown(KeyA); modifiers=[Control], action_dispatch=PathToolMessage::SelectAllPoints),
```

Comment:
You need to also declare that this action is active in the relevant context (likely in the Path tool). See the message in Discord where I gave you that hint (involving the `actions!(...)` macro) for more info. Once implemented, please build your code and open the editor to test that this PR's feature works and then set it to Ready for Review (instead of Draft) status and ping me again for another QA test. Lookin' forward to it :)